src/templates/EventList.html

```diff
@@ -1,5 +1,11 @@
-<div id="event-list">
-  <div v-for="event in events" class="event-card">
-    {{event.title}}
+<div id="event-list" class="event-list" v-show="view === 'list'">
+  {{ filteredEvents.length ? '' : '' }}
```

Comment:
Hmm, yeah that doesn't sound right to me. I'll try to take a look later today but have to be in a meeting now.